src/lib/uTop_complete.ml

```diff
@@ -892,6 +892,14 @@ and find_context_in_quotation = function
    | Completion                                                      |
    +-----------------------------------------------------------------+ *)
 
+#if OCAML_VERSION < (4, 11, 0)
+let longident_parse= Longident.parse
+#else
+let longident_parse str=
+  let lexbuf= Lexing.from_string str in
+  Parse.longident lexbuf
```

Comment:
I would propose to catch the syntax error here and return an option.

Reply:
Below the `complete` function, there is another `complete` function exactly share the same name that wraps and hides the former function with `try .. with`.

So we leave it unchanged?